Formula/raylib.rb

```diff
@@ -0,0 +1,38 @@
+class Raylib < Formula
+  desc "Simple and easy-to-use library to learn videogames programming"
+  homepage "http://www.raylib.com/"
+  url "https://github.com/raysan5/raylib/archive/1.9.1-dev.tar.gz"
+  version "1.9.1-dev"
+  sha256 "892357fb44d340eb7449c23c425d660d98b34b91434400e7610514ef02698600"
+
+  head "https://github.com/raysan5/raylib.git", :branch => "develop"
+
+  option "with-pic", "Compile static library as position-independent code"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".",
+      ("-DWITH_PIC=ON" if build.with? "pic"),
+      "-DSTATIC_RAYLIB=ON", "-DSHARED_RAYLIB=ON",
+      "-DMACOS_FATLIB=OFF", # Users should specify --universal if they want fat libraries
```

Comment:
comment isn't needed

Reply:
It serves as a hint that `--universal` has higher precedence than this CMake option?

Reply:
There is no universal option unless specifically added, and adding such an option is deprecated in Homebrew. So …

Reply:
Oh.. I thought `--universal` changes the `cmake_std_args`. What's the alternative? `--with-universal`?

Reply:
There is no alternative. We've fully deprecated `--universal` in homebrew/core.

Reply:
Removed.